app/controllers/autocomplete_controller.rb

```diff
@@ -91,16 +91,30 @@ def associated_tags
   def noncanonical_tag
     search_param = params[:term]
     raise "Redshirt: Attempted to constantize invalid class initialize noncanonical_tag #{params[:type].classify}" unless Tag::TYPES.include?(params[:type].classify)
+
     tag_class = params[:type].classify.constantize
-    render_output(tag_class.by_popularity
-                      .where(["canonical = 0 AND name LIKE ?",
-                              '%' + search_param + '%']).limit(10).map(&:name))
+    one_tag = tag_class.where(canonical: false, name: search_param)
+    # Is there a tag which is just right ( this is really for testing )
+    match = []
+    match = [one_tag.first.name] if one_tag.count == 1
```

Comment:
Is ES not systematically returning this tag with a high relevance?

Reply:
Put simply it is not guaranteed to and the test we have which is "Stargates SG-1" is an example were it does not work.